src/main/java/com/CezaryZal/entity/health/calendar/ConnectingUser.java

```diff
@@ -0,0 +1,15 @@
+package com.CezaryZal.entity.health.calendar;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ConnectingUser {
```

Comment:
co to za kolejny user?

Reply:
Tutaj miałem problem z nazwą.
Jest to encja pozwalająca stworzyć użytkownika w aplikacji HealthCalendar.
Żeby stworzyć użytkownika app HC potrzebuje tylko loginu i emaila.
Taka sama encja znajduje się również w aplikacji HC